src/main/package/www/languages/mc_locale_gui_source_en.json

```diff
@@ -40,7 +43,14 @@
   "FORWARD_PAYLOAD.DESTINATION_SENSOR" : "Destination Sensor",
   "FORWARD_PAYLOAD.DESTINATION_VARIABLE" : "Destination Variable",
   "FORWARD_PAYLOAD.FORWARD_PAYLOAD" : "Forward Payload",
+  "FORWARD_PAYLOAD.LAST_SEEN" : "Last seen:",
+  "FORWARD_PAYLOAD.MESSAGE_TYPE" : "Message Type:",
+  "FORWARD_PAYLOAD.NAME" : "Name:",
+  "FORWARD_PAYLOAD.NODE" : "Node:",
+  "FORWARD_PAYLOAD.NODE_ID" : "Node ID:",
+  "FORWARD_PAYLOAD.SENSOR" : "Sensor:",
```

Comment:
Kindly remove `:` from here and add it in html file, we have many occurrences with `:`

Example:
```
"FORWARD_PAYLOAD.LAST_SEEN" : "Last seen:",
 "FORWARD_PAYLOAD.MESSAGE_TYPE" : "Message Type:",
 "FORWARD_PAYLOAD.NAME" : "Name:",
 "FORWARD_PAYLOAD.NODE" : "Node:",
 "FORWARD_PAYLOAD.NODE_ID" : "Node ID:",
 "FORWARD_PAYLOAD.SENSOR" : "Sensor:",
```